plugins/discovery-ec2/src/test/java/org/elasticsearch/discovery/ec2/Ec2DiscoveryTests.java

```diff
@@ -233,4 +235,49 @@ public void filterByMultipleTags() throws InterruptedException {
         assertThat(discoveryNodes, hasSize(prodInstances));
     }
 
+    abstract class DummyEc2HostProvider extends AwsEc2UnicastHostsProvider {
+        public int fetchCount = 0;
+        public DummyEc2HostProvider(Settings settings, TransportService transportService, AwsEc2Service service, Version version) {
+            super(settings, transportService, service, version);
+        }
+    }
+
+    @Test
```

Comment:
Can you remove this annotation? It's not needed anymore and this will fail when you'll rebase on master.